apps/client/src/common/api/ontimeApi.ts

```diff
@@ -341,3 +341,27 @@ export const postPushSheet = async (id: string, options: ExcelImportMap) => {
   const response = await axios.post(`${ontimeURL}/sheet-push`, { id, options });
   return response.data.data;
 };
+
+/**
+ * @description HTTP request to rename a project file
+ */
+export async function renameProject(filename: string, newFilename: string): Promise<MessageResponse> {
+  const url = `${ontimeURL}/project/${filename}/rename`;
```

Comment:
question: did we check that this works well with names that contain spaces and other characters that we need to escape?

Reply:
it works for spaces and a few special chars I tested, it doesn't work for `/` as this causes issues with the path

Reply:
Sounds good, we can revisit this later. the validation of the name should make sure that the name is path friendly?